librarian/core/contrib/templates/decorators.py

```diff
@@ -8,7 +8,14 @@
 file that comes with the source code, or http://www.gnu.org/licenses/gpl.txt.
 """
 
-import functools
+
+class AttrDict(dict):
```

Comment:
Absolutely, and by the looks of it, `core.utils` should turn into a package itself